CHANGELOG.md

```diff
@@ -1,4 +1,31 @@
 
+# 0.11.0 - 2022-06-25
+
+The major change in this version is the increase of the Minimum Supported Rust Version (MSRV) from
+1.29 to 1.41.1, this is a big change because it introduces Rust Edition 2018 to the codebase along
```

Comment:
grammar nit: I think we should change `, this` to `. This`.

Reply:
Drats, I cut and pasted that line from the recent secp release notes. Will fix still as suggested.